README.md

```diff
@@ -1 +1,24 @@
-# parent_service_v1
\ No newline at end of file
+# parent_service_v1
+The Edustripe parent's service repository
+
+[![Build Status](https://travis-ci.com/edustripetech/parent_service_v1.svg?branch=develop)](https://travis-ci.com/edustripetech/parent_service_v1)  [![Coverage Status](https://coveralls.io/repos/github/edustripetech/parent_service_v1/badge.svg?branch=develop)](https://coveralls.io/github/edustripetech/parent_service_v1?branch=develop)  [![Maintainability](https://api.codeclimate.com/v1/badges/5b381e08c3a2331613a4/maintainability)](https://codeclimate.com/github/edustripetech/parent_service_v1/maintainability)  [![Test Coverage](https://api.codeclimate.com/v1/badges/5b381e08c3a2331613a4/test_coverage)](https://codeclimate.com/github/edustripetech/parent_service_v1/test_coverage)
+
+* [Technologies](#technologies)
+* [Getting Started](#getting-started)
+
+## Technologies
+
+- Node JS
+- Express
+- Babel
+
+## Getting Started
+
+- run `yarn` to install dependencies
+- create a .env file from the .env.example file `cp .env.example .env` and fill in the necessary environment variables
```

Comment:
Seems there is no `.env.example` in the project, guess you mean `.env.sample` here